pennylane/operation.py

```diff
@@ -232,10 +232,7 @@ class Operator(abc.ABC):
         wires (Sequence[int]): Subsystems it acts on. If not given, args[-1]
             is interpreted as wires.
         do_queue (bool): Indicates whether the operator should be
```

Comment:
Is `do_queue` needed any more? ever since we moved to having it False by default, I think it has been redundant. (Its original purpose was because queuing used to be _true_ by default, so we needed a way to turn it off).

Reply:
I guess also that it is redundant now. The only thing one could do is to actively block queuing when inside a QNode for example, but I don't see the use case